plugins/ui-update/utils/index.ts

```diff
@@ -49,4 +54,8 @@ function versionUpdateAvailable(
   return null;
 }
 
-export { versionUpdateAvailable };
+function getNotificationService(): NotificationService {
+  return container.get<NotificationService>(NotificationServiceType);
```

Comment:
I'm not sure that's a good way of using the injection. 
Why don't we create `Notificatior` class and have the service injected into it and then in hooks `plugins/ui-update/hooks.js` we actually use the container `container.get(Notificator); Notificator.setup()`. 
All names are fictional :)

Reply:
Ideally we should _not_ call container directly. But if we do we should try doing this at the top most level and let injection do everything else.

Reply:
I can move the get NS service to `hooks.js` and pass the Notification Service in as a parameter?

We'd have to talk about the `Notificatior` stuff I'm not quite grokking it from the comment.

Reply:
@nLight updated to get NotificationService from container in `hooks.js` and pass it as a parameter to `Notifications`. We can refactor to better use injection as an improvement. If thats ok with you.